docs/system/yocto/yocto-build-manually.md

```diff
@@ -45,13 +45,20 @@ repo sync --force-sync
 
 Set up environment variables:
 ```bash
-export DIR_WORK=$PWD/../
+export DIR_WORK=$PWD
 export BUILD_TAG=*optional build Tag*
 export PLATFORM_VERSION="$(git -C $DIR_WORK.repo/manifests rev-parse --short HEAD)"
 export PLATFORM_VERSION_DETAILS="$(repo forall -c 'echo $REPO_PATH\nLREV: $REPO_LREV\nRREV: $REPO_RREV; git diff --stat -b $REPO_LREV..HEAD ; echo -n "Commit: " ; git rev-parse HEAD ; echo -n "Uncommited changes: " ; git status -b -s ; git diff --stat -b ; echo ')"
 echo "building with repo versions: $PLATFORM_VERSION"
 export BB_ENV_PASSTHROUGH_ADDITIONS="$BB_ENV_PASSTHROUGH_ADDITIONS BUILD_TAG PLATFORM_VERSION PLATFORM_VERSION_DETAILS"
-export TEMPLATECONF=$PWD/../sources/meta-mobility-poky-distro/conf
+#old before kirkstone export TEMPLATECONF=$PWD/sources/meta-mobility-poky-distro/conf
+export TEMPLATECONF=$PWD/sources/meta-mobility-poky-distro/conf/generic
```

Comment:
How about adding a comment right before explaining the two options generic and hmx?

Reply:
maybe this is better.
#if unsure what is supported see meta-mobility-poky-distro/conf/generic or hmx/ conf-notes.txt for more information to select correct path for your MACHINE build.